css/index.css

```diff
@@ -143,4 +142,192 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+/* Start home */
+.logo {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-items: baseline;
+    align-content: flex-start;
+  }
+
+.logo-img {
+    flex-basis: 100%;
+    order: 2;
+}
+
+header {
+    width: 100%;
+    background: white;
+}
+header nav {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 20px;
+    font-size: 14px;
+}
+
+header nav a {
+    color: black;
+    text-decoration: none;
+}
```

Comment:
The `.middle-content .boxes .box {
    width: 12.5%;
    height: 100px;
    margin: 20px 2.5%;
    color: white;
    display: flex;
    align-items: center;
    justify-content: center;
}` 
contains all the styles you need for each box.  The only thing that needs to be done in individual boxes is overriding the color of the box.